csrankings.csv

```diff
@@ -16153,3 +16153,37 @@ François Fleuret,EPFL,https://people.epfl.ch/francois.fleuret,Bj1tRlsAAAAJ
 Alexandre Alahi,EPFL,https://people.epfl.ch/alexandre.alahi,UIhXQ64AAAAJ
 Pierre Vandergheynst,EPFL,https://people.epfl.ch/pierre.vandergheynst,1p9NOFEAAAAJ
 Ali H. Sayed,EPFL,https://people.epfl.ch/ali.sayed,uDG9sXQAAAAJ
+Christian S. Jensen,Aalborg University,http://people.cs.aau.dk/~csj/,wo20cPYAAAAJ
+Kim G. Larsen,Aalborg University,www.cs.aau.dk/~kgl,neDFD60AAAAJ

```

Comment:
please make a proper URL

Reply:
The URL problem is addressed in the latest commit.